hudi-common/src/main/java/org/apache/hudi/common/fs/StorageSchemes.java

```diff
@@ -28,6 +28,8 @@
   FILE("file", false),
   // Hadoop File System
   HDFS("hdfs", true),
+  // Baidu Advanced File System
+  AFS("afs", true),
```

Comment:
supports append? nice!

Reply:
Yes. Baidu: the biggest search engine in China.

Reply:
@xglv1985  Would you please also add an entry here http://hudi.apache.org/docs/cloud.html and a page like this http://hudi.apache.org/docs/ibm_cos_hoodie.html to describe how to use AFS?

Reply:
> @xglv1985 Would you please also add an entry here http://hudi.apache.org/docs/cloud.html and a page like this http://hudi.apache.org/docs/ibm_cos_hoodie.html to describe how to use AFS?

Hello, Baidu AFS is currently just open to its staff. As if the way to use afs, it is highly similar to HDFS. Actually, AFS 
do some optimization from the base level, and does not change the main user api. The users can not perceive their difference. But if the document is mandatory, could I update the doc once AFS is open to all users?

Reply:
OK, sounds good.